js/components/userbar/user-bar.js

```diff
@@ -46,11 +46,13 @@ define([
 				jobListing: ko.computed(() => lodash.sortBy(this.jobListing(), el => -1 * el.executionId)
 					.filter( j => j.ownerType === constants.jobTypes.USER_JOB.ownerType)),
 				jobNameClick: this.jobNameClick.bind(this),
+				showAsHyperlinks: true,
 			};
 			this.allJobParams = {
 				jobListing: ko.computed(() => lodash.sortBy(this.jobListing(), el => -1 * el.executionId)
 					.filter( j => j.ownerType === constants.jobTypes.ALL_JOB.ownerType)),
 				jobNameClick: this.jobNameClick.bind(this),
```

Comment:
You do not need "jobNameClick" parameter if "showAsHyperlinks" is false. Maybe even use it as flag instead of additional parameter "showAsHyperlinks"?